doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+   You can identify the nRF9160 DK by the fact that it has three COM ports.
+
+   .. figure:: /images/programmer_com_ports.png
+      :alt: Programmer - COM ports
+
+      Programmer - COM ports
+
+   If the three COM ports are not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
+
+   The button text changes to the type of the selected device, with the SEGGER ID below the name.
+   The :guilabel:`Device Memory Layout` section also changes its name to the device name, and indicates that the device is connected.
+   If the :guilabel:`Auto read memory` option is selected in the :guilabel:`Device` section ofthe side panel, the memory layout will update.
+   If it is not selected and you wish to see the memory layout, click :guilabel:`Read` in the :guilabel:`Device` section of the side panel.
+
+#. Click :guilabel:`Add file` in the :guilabel:`File` section of the side panel.
+
+   .. figure:: /images/programmer_addfile_nrf9160dk.png
+      :alt: Programmer - Add file
+
+      Programmer - Add file
+
+#. Navigate to where you extracted the firmware, and choose the :file:`mfwnrf9160_<version-number>.zip` file.
+#. Click :guilabel:`Write` in the :guilabel:`Device` section of the side panel.
+
+   .. figure:: /images/programmer_write_nrf9160dk.png
+      :alt: Programmer - Write
+
+      Programmer - Write
+
+   The :guilabel:`Modem DFU` dialog window appears.
+
+   .. figure:: images/programmerapp_modemdfu.png
+      :alt: Modem DFU window
+
+      The Modem DFU window
+
+#. Click the :guilabel:`Write` button in the :guilabel:`Modem DFU` dialog window to update the firmware.
+   Do not unplug or turn off the device during this process.
+
+When the update is complete, you see a success message.
+
+If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (Cmd+R on macOS), and try again.
+
+.. note::
+
+   If you experience issues updating the modem firmware, click :guilabel:`Erase all` in the :guilabel:`Device` section of the side panel and try updating again.
+
+   Doing this erases the contents of the flash memory, so the application must programmed again to the DK.
+
+.. _nrf9160_gs_updating_cloud_certificate:
+
+Updating the nRF Cloud certificate
+**********************************
+
+To test the nRF9160 DK with the :ref:`asset_tracker_v2` application, you need an `nRF Cloud`_ account.
+nRF Cloud is Nordic Semiconductor's platform for connecting your IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor's chips, and more.
+It also includes a public REST Application Programming Interface (API) that can be used for building your own IoT solutions.
+
+Since the application transmits data to nRF Cloud for visualization, it requires a valid security certificate.
+This means that you might need to update the certificate stored on the DK.
```

Comment:
This is to my knowledge only necessary when use applications that talk to the REST api. Like the location sample.
It is a quite complicated process to do this step. I am fine with making it mandatory, but just wanted to inform you that it is not needed for testing the asset tracker v2

Reply:
@simensrostad correct me if I am wrong

Reply:
Does this mean that it is enough to keep the Creating an nRF Cloud account (probably moved under Connecting the DK to nRF Cloud), and then move the rest of Updating the nRF Cloud certificate to be inside working with nRF9160 DK?

Reply:
Yes, I think so, I'd say that a link here to how to generate new certificates should be sufficient.

Reply:
Moved the sections as discussed.
Is this good?

Reply:
Yes, looks good